ansible/roles/operator-pipeline/templates/openshift/pipelines/operator-ci-pipeline.yml

```diff
@@ -41,15 +41,19 @@ spec:
     - name: pipeline_image
       description: An image of operator-pipeline-images.
       default: "quay.io/redhat-isv/operator-pipelines-images:v1.0.7"
+    - name: pyxis_api_key_secret_name
+      description: Kubernetes secret name that contains Pyxis API key. Valid only when external Pyxis is used.
+      default: pyxis-api-secret
```

Comment:
The default values of `pyxis_api_key_secret_name` and `pyxis_api_key_secret_key` can't be empty strings, because there are specific naming requirements for secret names that an empty string can't fulfill, and would cause an error.